_maps/RandomRuins/IceRuins/icemoon_underground_hermit.dmm

```diff
@@ -0,0 +1,452 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"bz" = (
+/obj/item/flashlight/lantern,
+/turf/open/floor/grass/fairy,
+/area/ruin/powered/shuttle)
+"cI" = (
+/turf/open/floor/plating,
+/area/ruin/powered/shuttle)
+"de" = (
+/obj/item/flashlight/lantern,
+/turf/open/floor/plating/asteroid/snow/icemoon,
+/area/icemoon/underground/explored)
+"ec" = (
+/obj/item/clothing/suit/hooded/wintercoat,
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"ei" = (
+/obj/machinery/hydroponics/soil,
+/turf/open/floor/grass/fairy,
+/area/ruin/powered/shuttle)
+"gr" = (
+/turf/open/floor/grass/fairy,
+/area/ruin/powered/shuttle)
+"ha" = (
+/obj/machinery/light{
+	dir = 8
+	},
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"hv" = (
+/obj/structure/bonfire/prelit,
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"hN" = (
+/turf/open/floor/wood,
+/area/ruin/powered/shuttle)
+"ii" = (
+/obj/effect/mob_spawn/human/hermit{
+	flavour_text = "You've been living on the ice moon for at least 40 years now! You've survived the ice whelps, hunted those damn bears and tamed a few wolves. What do those outsiders want?";
```

Comment:
if the hermit's supposed to have tamed a few wolves, can you add a few friendly wolves to his shack?

Reply:
I'm not entirely sure we have any tame wolf subtypes.

Reply:
just make one its easy

Reply:
I don't really know how, and learning to honour a tiny bit of fluff text seems a bit much.

Reply:
```
/mob/living/simple_animal/hostile/asteroid/wolf/tame
	name = "Tamed Wolf"
	faction = list("neutral")
```
bam, done

hell, you don't even have to make a new subtype for it, you could just override variables like you've done here with the hermit's flavour_text

Reply:
Add these var edits to the object definition. Poke on the coder channel if you want to know how to.